data/korath/korath outfits.txt

```diff
@@ -178,6 +205,241 @@ outfit "Control Transceiver"
+	description "Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."
+	description "Adding these afterburners to an interceptor allows it to reach absurd speeds; however, the heat produced ensures that running them for any substantial period of time will require some serious cooling."
+
+outfit "Afterburner (Lunar Class)"
+	plural "Afterburners (Lunar Class)"
+	category "Engines"
+	"cost" 647000
+	thumbnail "outfit/medium korath afterburner"
+	"mass" 32
+	"outfit space" -32
+	"engine capacity" -32
+	"afterburner thrust" 148.8
+	"afterburner energy" 3.92
+	"afterburner heat" 41.65
+	"afterburner fuel" 0.18
+	"afterburner effect" "korath afterburner"
+	description "Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."
+	description "Although these afterburners produce an incredible amount of thrust for their fuel usage, you can't help but feel concerned about the safety of installing one."
+
+outfit "Afterburner (Planetary Class)"
+	plural "Afterburners (Planetary Class)"
+	category "Engines"
+	"cost" 1459000
+	thumbnail "outfit/large korath afterburner"
+	"mass" 55
+	"outfit space" -55
+	"engine capacity" -55
+	"afterburner thrust" 288
+	"afterburner energy" 7.12
+	"afterburner heat" 79.45
+	"afterburner fuel" 0.32
+	"afterburner effect" "korath afterburner"
+	"heat capacity" 500
+	"heat dissipation" 0.02
+	description "Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."
+	description "These enormous afterburners have dedicated reaction chambers with cooling vents to keep the fuel-plasma explosion from destroying the thruster."
+
+outfit "Afterburner (Stellar Class)"
+	plural "Afterburners (Stellar Class)"
+	category "Engines"
+	"cost" 3280000
+	thumbnail "outfit/huge korath afterburner"
+	"mass" 94
+	"outfit space" -94
+	"engine capacity" -94
+	"afterburner thrust" 552
+	"afterburner energy" 12.64
+	"afterburner heat" 148
+	"afterburner fuel" 0.58
+	"afterburner effect" "korath afterburner"
+	"heat capacity" 900
+	"cooling energy" 0.4
+	"active cooling" 8
+	"flare sound" "plasma medium"
+	description "Korath afterburners pump super-heated hyperspace fuel directly into a thruster's plasma stream, resulting in a phenomenal explosion that nearly melts the thruster."
+	description "The very largest of Korath afterburners have integrated active cooling systems, powered by the fuel-plasma explosion, in order to keep the reaction chambers from melting."
+
+effect "korath afterburner"
+	sprite "effect/korath afterburner"
+		"no repeat"
+		"frame rate" 55
+	sound "korath afterburner"
+	"lifetime" 10
+	"random angle" 15
+	"random spin" 360
+	"random frame rate" 5
+	"random velocity" 1.
+	"velocity scale" 1
```

Comment:
Same with this as the Heaver thing. It's a bit out of scope for the current PR as it's content-frozen for additions like this and review ready.

However, if Raven wants to do this, we don't need any new code or anything, just different afterburner effects using a different ```scale```.

Reply:
No, that's a great suggestion.  If `scale` will do the trick I'll certainly do that rather than make more sprites though, as it's less of a change for this late stage.

Reply:
Scale should do it, I would recommend not scaling up too much, but scaling down should be nice and safe.